.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: kind/triage
+assignees: ''
+
+---
+
+#### Problem Description
+A clear and concise description of what the bug is.
+
+#### Steps to reproduce the behavior:
+1. 
+2. 
+3. 
+
+#### System Information
+Paste the output of "mitmproxy --version" here.
```

Comment:
How about a few more?
OS, uname -a, lsb_release -sc, ...
pip3 -V
How did you install mitmproxy?

Reply:
`mitmproxy --version` already provides a lot:
```
Mitmproxy: 5.0.0.dev (+404, commit 7bf70da)
Python:    3.8.0
OpenSSL:   OpenSSL 1.1.1d  10 Sep 2019
Platform:  Linux-4.4.0-18362-Microsoft-x86_64-with-glibc2.27
```
I think that's good enough for 99% of issues. If that's your only concern I'd go ahead here? :)